### ISSUE TITLE
Package subpath './dist/esm/config/get-nestjs-webpack-prod.config' is not defined by "exports"

### ISSUE THREAD

**Issue by vuletuanbt** [rnw-community/rnw-community#105]
I install `@rnw-community/nestjs-webpack-swc` and follow steps to setup. When i run `nest build --webpack --webpackPath webpack-prod.config.js ` then error occur

node: v16.14.2
```json
"@nestjs/cli": "^8.0.0",
"@rnw-community/nestjs-webpack-swc": "^0.43.3",
"@rnw-community/shared": "^0.43.2",
"@swc/cli": "^0.1.57",
"@swc/core": "^1.3.4",
"@swc/helpers": "^0.4.11",
"swc-loader": "^0.2.3",
```

**Comment by vitalyiegorov** [comment]
@vuletuanbt Thanks for the report! fixed in #106

**Comment by vuletuanbt** [comment]
> @vuletuanbt Thanks for the report! fixed in #106, also the docs are updated, please updated your local js scripts and change the imports.

thanks for your quick support 🙇🏽‍♂️

**Comment by vitalyiegorov** [comment]
@vuletuanbt Feel free to give any feedback regarding this package, or other packages that this monorepo provides.
